src/tools/TerminalStress/TerminalStress.csproj

```diff
@@ -2,8 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net6.0</TargetFramework>
-    <RuntimeFrameworkVersion>6.0.9</RuntimeFrameworkVersion>
```

Comment:
we keep this pinned to a specific version so that, when the .NET team releases a new minor point release, our builds don't instantly fall over because our package feed doesn't contain `8.0.124` instead of `8.0.123`.

Please pin the latest 8.x version :)

Reply:
The docs say:
> For self-contained deployments, the `RuntimeFrameworkVersion` specifies the exact runtime framework version. For framework-dependent applications, the `RuntimeFrameworkVersion` specifies the minimum required runtime framework version.

Which one are we? I would've naively assumed that we're framework-dependent.

Reply:
huh. you're right.